geometry_msgs/Pose[] face_det
geometry_msgs/Pose[] body_det
geometry_msgs/Pose2D leg_det
string label  						
---
detect_human[] detect_human
